src/components/autocomplete/autocomplete.stories.js

```diff
@@ -0,0 +1,140 @@
+import { html } from "lit-html";
+import "../autocomplete";
+import { docsIntro } from "../../../.storybook/utils.js";
+
+export default {
+  title: 'custom-elements/vl-autocomplete',
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'autocomplete',
+          intro: 'Use autocomplete to show ...',
+        }),
+      },
+    },
+  },
+};
+
+//--------------------------
+
+export const Default = () => html`
+  <vl-autocomplete data-min-chars="1" 
```

Comment:
Attributen schrijven we (voor consistentie) altijd als `data-vl-` en dan de naam van het attribuut.
Dat pas je best even overal voor je attributen aan.

Reply:
attributen aangepast zodat ze beginnen met data-vl in branch